core/src/test/java/de/evoila/cf/security/utils/RandomStringTest.java

```diff
@@ -0,0 +1,118 @@
+        when(mockRandom.nextInt(52))
+                .thenAnswer(new Answer() {
+                                int index = 0;
+                                @Override
+                                public Object answer(InvocationOnMock invocationOnMock) {
+                                    if (index == 52) {
+                                        index = 0;
+                                    }
+                                    return index++;
+                                }
+                            }
+                );
+        RandomString randomString = new RandomString(52, mockRandom, false,false);
+        String string = randomString.nextString();
+        assertEquals(string.length(), 52);
+        assertEquals("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz", string);
+
+    }
+
+    @Test
+    void RandomString_LowerCaseAndNumbers() {
+        when(mockRandom.nextInt(62))
+                .thenAnswer(new Answer() {
+                                int index = 0;
+                                @Override
+                                public Object answer(InvocationOnMock invocationOnMock) {
+                                    if (index == 62) {
+                                        index = 0;
+                                    }
+                                    return index++;
+                                }
+                            }
+                );
+        RandomString randomString = new RandomString(62, mockRandom, true,true);
+        String string = randomString.nextString();
+        assertEquals(string.length(), 62);
+        assertEquals("abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz0123456789", string);
+
+    }
+
+    @Test
+    void RandomString_LowerAndUpperCaseAndNumbers() {
+        when(mockRandom.nextInt(62))
+                .thenAnswer(new Answer() {
+                                int index = 0;
+                                @Override
+                                public Object answer(InvocationOnMock invocationOnMock) {
+                                    if (index == 62) {
+                                        index = 0;
+                                    }
+                                    return index++;
+                                }
+                            }
+                );
+        RandomString randomString = new RandomString(62, mockRandom, true,false);
+        String string = randomString.nextString();
+        assertEquals(string.length(), 62);
+        assertEquals("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789", string);
+
+    }
+
+    @Test
+    void RandomString_lengthTooSmall() {
+        assertThrows(IllegalArgumentException.class, () -> new RandomString(0));
+    }
+
+    @Test
+    void RandomString_lengthNegativ() {
+        assertThrows(IllegalArgumentException.class, () -> new RandomString(-3));
+    }
+
+    @Test
+    void RandomString_randomNull() {
```

Comment:
Please do not use underscores in method names. In Java the camel case notation is commonly used for naming.